test/tlv/TlvSchemaTest.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2022 Project CHIP Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TlvObject, TlvField } from "../../src/tlv/TlvObject.js";
+import { TlvString } from "../../src/tlv/TlvString.js";
+import { TlvBoolean } from "../../src/tlv/TlvBoolean.js";
+import { TlvArray } from "../../src/tlv/TlvArray.js";
+import { 
+    TlvUInt8, TlvUInt16, TlvUInt32, TlvUInt64, 
+    TlvInt8, TlvInt16, TlvInt32, TlvInt64,
+    TlvFloat, TlvDouble 
+} from "../../src/tlv/TlvNumber.js";
+
+import { strict as assert } from 'assert'
```

Comment:
Assert is not listed as a dependencies in package.json so this works, it means that is is imported indirectly by another dependency.
However, it would be better to avoid depending on indirect dependencies, since version can change or it can disappear...

I would recommend to use the jest / jasmine builtin expect instead:
assert.equal(a, b) => expect(a).toBe(b)

Reply:
I would like an option to use `assert` as it tends to be more portable across different testing frameworks compared to a `jest` specific option.  Is there a problem using `assert` for the browser test framework?

Reply:
Ok.  Converted to jest format.